cv19index/resources/xgboost/claims.schema.json

```diff
@@ -0,0 +1,132 @@
+{
+  "schema": [
```

Comment:
Not sure why we have these.  We shouldn't be requiring people to set up a schema.json file for their inputs.  They shoudl be able to just send in a file.

Reply:
The IO parser uses them to get the needed column set. I figured they could be used to outline what columns you did need?

Reply:
Ah, so this is just the "needed columns".  They can have whatever other columns they want and it's ok?  That's fine.

Reply:
Yes, that's correct. These are the required columns, extra columns will not fail processing.